src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -66,8 +66,19 @@
                               :onClick #(swap! state update :expanded not)})))
         [:div {:style {:padding "0.25em 0 0 0"} :id "chart_div"}]
         (if (:expanded @state)
-          (.timingDiagram js/window (:data props) (:workflow-name props))
-          (when (.getElementById js/document "chart_div") (gdom/remove-children "chart_div")))])})
+          (.timingDiagram js/window (:data props) (:workflow-name props) nil)
+          (when (.getElementById js/document "chart_div") (gdom/remove-children "chart_div")))])
```

Comment:
Document IDs must be globally unique, but you have no way to enforce that here. Use a ref instead. Also, don't manipulate children inside of the render method. Move all children modifications to `component-did-mount` or `component-did-update`.